way2sms-api.php

```diff
@@ -21,12 +21,13 @@ function sendWay2SMS($uid, $pwd, $phone, $msg)
   $uid = urlencode($uid);
   $pwd = urlencode($pwd);
 
-  $autobalancer = rand(1, 8);
+  //$autobalancer = rand(1, 8);
+  $autobalancer = 23;
```

Comment:
This will lead to all the request's going to one of way2sms server, and if that server is loaded, this may cause sms to be delivered late. 

Change this back to a random range of valid way2sms servers :)